django-verdant/rca/static/rca/css/components/hero.less

```diff
@@ -0,0 +1,130 @@
+.hero {
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin: 0 -@spacing-one;
+    margin-bottom: @spacing-onepointfive;
+    padding: 0 @spacing-one;
+
+    @media @media-desktop-small {
+        margin-right: 0;
+        margin-left: 0;
+        padding: 0;
+    }
+
+    &__inner {
+        padding-bottom: @spacing-one;
+    }
+
+    &__text {
+        color: @color-white;
+        font-size: @font-size-24;
+        max-width: 745px;
+    }
+
+    &__title-wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-end;
+        margin-bottom: @spacing-one;
+    }
+
+    &__title {
+        color: @color-white;
+        font-size: 80px;
+        font-family: @font-family-knockout;
+        font-weight: @font-weight-knockout;
+        text-transform: uppercase;
+        line-height: 0.8;
+        margin: 0;
+    }
+
+    &__browse-link {
+        display: block;
+        color: @color-white;
+        font-family: @font-family-benton;
+        font-size: 15px;
+        text-transform: uppercase;
+        display: flex;
+        align-items: center;
+    }
+
+    &__browse-text {
+        max-width: 120px;
+        text-align: right;
+
+        &--text-alt {
+            color: lighten(@color-highlight-blue, 25%);
+        }
+    }
+
+    &__browse-icon {
+        width: 40px;
+        height: 40px;
+        fill: lighten(@color-highlight-blue, 25%);
+        margin-left: 10px;
+    }
+
+    &__search-wrapper {
+        position: relative;
+    }
+
+    &__search-input {
+        background: @color-white;
+        padding-right: 50px;
+
+        &::placeholder {
+            color: @color-highlight-blue;
+            font-family: @font-family-benton;
+        }
+    }
+
+    &__search-icon {
+        width: 22px;
+        height: 22px;
+        fill: @color-highlight-blue;
+        position: absolute;
+        right: 15px;
+        top: 50%;
+        transform: translateY(-50%);
+    }
+}
+
+.template-home-page {
+    .ui-menu {
+        background: @color-black;
+        display: flex;
+        flex-wrap: wrap;
+        // Important to override js
+        // 100% width minus side padding
+        max-width: calc(100vw - 30px) !important;
```

Comment:
from a quick test not certain the `!important` is needed here - the js sets `width` as an inline style, but can't see `max-width`?